File: multilibrary/src/main/java/com/zhongjh/albumcamerarecorder/camera/listener/EditListener.java
package com.zhongjh.albumcamerarecorder.camera.listener;

import android.net.Uri;

/**
 * 编辑事件
 * @author zhongjh
 */
public interface EditListener {

    /**
     * 编辑图片
     *
     * @param uri 当前需要编辑图片
     * @param newPath 编辑后的图片path
     */
    void onImageEdit(Uri uri, String newPath);

}

File: multilibrary/src/main/java/com/zhongjh/albumcamerarecorder/camera/listener/EditListener.kt
package com.zhongjh.albumcamerarecorder.camera.listener

import android.net.Uri

/**
 * 编辑事件
 * @author zhongjh
 */
interface EditListener {
    /**
     * 编辑图片
     *
     * @param uri 当前需要编辑图片
     * @param newPath 编辑后的图片path
     */
    fun onImageEdit(uri: Uri, newPath: String)
}